src/keystore.js

```diff
@@ -32,105 +29,63 @@ type HDPrivateKey = {|
   |},
 |}
 
-type PasswordOptionsUser = {|
-  scryptParams?: ScryptParams,
-  encryptionType?: string,
-  saltBytesCount?: number,
-  derivedKeyLength?: number,
-|}
-
-type MnemonicOptionsUser = {|
-  network?: ?Network,
-  passphrase?: ?string,
-  derivationPath?: string,
-  paddedMnemonicLength?: number,
-|}
-
 const DEFAULT_SCRYPT_PARAMS: ScryptParams = {
   N: 2 ** 18,
   r: 8,
   p: 1,
 }
 
 const DEFAULT_NETWORK: string = 'livenet'
-const ADDRESS_TYPE: 'address' = 'address'
-const MNEMONIC_TYPE: 'mnemonic' = 'mnemonic'
-const CURRENT_VERSION: string = packageData.version
 const DEFAULT_ENCRYPTION_TYPE: string = 'nacl.secretbox'
 const DEFAULT_DERIVATION_PATH: string = 'm/44\'/60\'/0\'/0'
 const PADDED_MNEMONIC_LENGTH: number = 120
 const DEFAULT_SALT_BYTES_COUNT: number = 32
 const DEFAULT_DERIVATION_KEY_LENGTH: number = 32
 
-function checkAddressValid(address: string): boolean {
-  return utils.checkAddressValid(address)
-}
-
-function checkChecksumAddressValid(address: string): boolean {
-  return utils.checkChecksumAddressValid(address)
+export {
```

Comment:
It was possible to use
```
export { testPassword } from './password'
export { generateMnemonic, checkMnemonicValid, checkBip32XPublicKeyValid } from './mnemonic'

```
instead of

```
import { X } from 'xxxx'
export { X }
```